Automattic/wp-calypso
# Importers: Remove the GoDaddy GoCentral importer

It has been concluded that the import is no longer reliable and therefore we remove it as an option from Calypso

#### Changes proposed in this Pull Request

- This removes the GoDaddy importer from the calypso importer list.
- It also removes references to the importer from the importer signup flow (which will only support wix).


#### Testing instructions

– Click on Tools/Import in the sidebar and make sure that the GoDaddy importer is not listed.
